cclient/api/extended/get_new_address.c

```diff
@@ -42,7 +40,16 @@ retcode_t is_unused_address(iota_client_service_t const* const serv, flex_trit_t
     goto done;
   }
 
+done:
+  were_addresses_spent_from_req_free(&addr_spent_req);
+  were_addresses_spent_from_res_free(&addr_spent_res);
+  return ret_code;
+}
+
+static retcode_t find_transactions_by_address(iota_client_service_t const* const serv, flex_trit_t const* const addr,
+                                              bool* const is_unused, bool get_tx_list, hash243_queue_t* txs) {
```

Comment:
Do we need the transactions ordered ? Maybe a list/stack instead of a queue ?

Reply:
not sure yet, could be changed later.